arenadata/docker-compose.yaml

```diff
@@ -2,21 +2,22 @@
 
 version: "3"
 services:
-  #image can be either hub.adsw.io/library/gpdb6_regress:${BRANCH_NAME}-x86-64 or
-  #hub.adsw.io/library/gpdb6_regress:${BRANCH_NAME}-ppc64le depending on the arch
-  mdw:
+  #image can be either hub.adsw.io/library/gpdb8_regress:${BRANCH_NAME}-x86-64 or
+  #hub.adsw.io/library/gpdb8_regress:${BRANCH_NAME}-ppc64le depending on the arch
```

Comment:
Is that true? I think we haven't supported the `ppc64le` architecture for a long time.

Reply:
fixed